search-service/src/main/kotlin/com/egm/stellio/search/temporal/util/TemporalQueryUtils.kt

```diff
@@ -45,18 +45,22 @@ fun composeTemporalEntitiesQueryFromGet(
 
     if (inQueryEntities)
         entitiesQueryFromGet.validateMinimalQueryEntitiesParameters().bind()
+    val optionsParam = Optional.ofNullable(requestParams.getFirst(QueryParameter.OPTIONS.key))
+    val formatParam = Optional.ofNullable(requestParams.getFirst(QueryParameter.FORMAT.key))
+    val withTemporalValues = when {
```

Comment:
you could take this opportunity to return an `APIException` if both `temporalValues` and `aggregatedValues` are provided